routes/filter.go

```diff
@@ -208,10 +210,15 @@ func (r *filterHandler) Get(c *gin.Context) {
 
 func (r *filterHandler) SetRoutes(router *gin.Engine) {
 
-	filterRouter := router.Group("/filter")
-	{
-		filterRouter.GET("/:resource", r.Get)
-	}
+	projectRouter := router.Group("/project")
```

Comment:
It's odd only project and asset are in singular form. But I realise they are singular in current version.